test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -190,7 +190,8 @@ public URL getResource(String name) {
 
                                         @Override
                                         protected ClassLoader getClassLoader() {
-                                            return temp;
```

Comment:
Do we still need this class? Can it just use QuarkusClassWriter?

Reply:
Yeah we do unfortunately need to override the method here because the TCCL at the time this executed has been changed to `temp`.
On the other hand if you are asking if we absolutely need `QuarkusClassWriter` since we aren't actually using it's method, the answer is no, we don't, we could just use a regular `ClassWriter`. But I think keeping it around will make finding the transformation usages easier since all you have to do is see where `QuarkusClassWriter` is being used. 
WDYT?